azure-pipelines-woa.yml

```diff
@@ -78,7 +78,7 @@ steps:
   displayName: 'Verify ffmpeg'
 
 - script: |
-    cd src
-    echo "Verifying mksnapshot"
-    python electron\script\verify-mksnapshot.py --build-dir out\Default --source-root %cd%
-  displayName: 'Verify mksnapshot'
+    taskkill /F /IM electron.exe
+    taskkill /F /IM MicrosoftEdge.exe
```

Comment:
So the processes don't exit cleanly, is this an issue ? Also why is there a edge process running in ci ?

Reply:
I think the edge process is because of the `shell.openExternal` test where we open a page in the default browser.

Reply:
@deepak1556 those lines are in there to handle bad builds.  The tests don't run in a container, so we need to make sure that the OS is cleaned up for the next test run.   @brenca is right about the edge process running as a result of `shell.openExternal`.

Reply:
@deepak1556 those lines are in there to handle bad builds. The tests don't run in a container, so we need to make sure that the OS is cleaned up for the next test run. @brenca is right about the edge process running as a result of shell.openExternal.